### PR TITLE
Basic F# Scaffolding for Suave

### PR DESCRIPTION
**Pull request by drusellers**

### CODE REVIEW

**Comment by dahlbyk** [comment]
Can you clean up the several references to `hucache`?

**Comment by dahlbyk** [comment]
Added a `.gitattributes` for great justice

**Comment by drusellers** [comment]
> Can you clean up the several references to hucache?

check now

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.fake/
+.paket/
```
Comment (dahlbyk) [inline .gitignore]: I think we need this for build scripts to have what they need.

**Pull request merged**